recipe/meta.yaml

```diff
@@ -1,14 +1,15 @@
-{% set version = "3.0.0b3" %}
-{% set version_str = "v3.0.0-beta-3" %}
+{% set version = "3.0.0" %}
+{% set version_str = "v3.0.0" %}
+{% set sha256 = "f5b3563f118f1d3d6e001705fa7082e8fc3bda50038ac3dff787650795734146" %}
 
 package:
     name: protobuf
     version: {{ version }}
 
 source:
-    fn: protobuf_{{ version }}.tar.gz
+    fn: protobuf-{{ version }}.tar.gz
     url: https://github.com/google/protobuf/archive/{{ version_str }}/protobuf-{{ version_str }}.tar.gz
-    md5: 89afd3855f2d4782e59c09e07d9efa67
+    sha256: {{ sha256 }}
 
 build:
     number: 0
```

Comment:
In `master`, it is [`1`]( https://github.com/conda-forge/protobuf-feedstock/blob/91c9b2c304a98bd4221815ae4649f14ff7ed8124/recipe/meta.yaml#L14 ). Not sure why we are not able to see it here. Though I do know we need to rebase/merge and then force this back to `0`.